corehq/apps/domain/forms.py

```diff
@@ -2008,7 +2013,7 @@ def send_prepayment_invoice(self, date_start, date_end):
         )
 
     @property
-    def is_annual_plan(self):
+    def selected_annual_plan(self):
```

Comment:
`is_annual_plan_selected` might be better?

Reply:
1fec17a